pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -315,6 +315,19 @@ func (k *Kubernetes) UpdateKubernetesObjects(name string, service kobject.Servic
 
 	// Configure the container volumes.
 	volumesMount, volumes, pvc := k.ConfigVolumes(name, service)
+	// Configure Tmpfs
+	if len(service.TmpFs) > 0 {
```

Comment:
I think this is better than just `!=nil`

if you test only for `!=nil` than it would be `false` if `service.Tmpfs` is empty array.

https://play.golang.org/p/9W5KUrr9jY